custom_components/luxtronik/const.py

```diff
@@ -350,6 +350,12 @@ class LuxMode(Enum):
         icon="mdi:waves-arrow-right",
         sensor_key="calculations.ID_WEB_Temperatur_TRL",
     ),
+    LuxtronikSensorEntityDescription(
+        key="flow_out_temperature",
```

Comment:
This key is already used for "calculations.ID_WEB_Temperatur_TRL".
Please change it.